majifix311/src/test/java/com/example/majifix311/ProblemListActivityTest.java

```diff
@@ -97,7 +97,7 @@ private void sendMocks() {
         mockProblems.add(partialProblem1);
         mockProblems.add(partialProblem2);
 
-        EventHandler.sendMyReportedProblemsList(RuntimeEnvironment.application, mockProblems);
+        EventHandler.retrievedMyRequests(RuntimeEnvironment.application, mockProblems, false);
```

Comment:
At some point, this will need to be renamed to accommodate two  calls - one for "my reported problems" and one for "my assigned problems".